rfc/github-cdevents.md

```diff
@@ -0,0 +1,58 @@
+## Github-CDEvents Integration
+
+### Overview
+The integration of CDEvents with Github is made by converting Github events into CDEvents. This effort uses Github Webhook generated events and translate them into CDEvents.
+
+Github utilizes two types of events: the github-event and the issue-event. These events are to be converted into [CDEvents-Source Code Control Events](https://github.com/cdevents/spec/blob/v0.3.0/source-code-version-control.md) 
+using a custom github-event to CDEvent translator. 
+
+### Github webhook
+On github, there are different types of webhooks: organization, repository, marketplace, sponsors and app webhook. 
+- Organization webhooks can subscribe to events that happen in all repositories owned by the organization. They can also subscribe to events that happen at the organization level that are outside of any particular repository.
+- Marketplace webhook is the one that subscribes to events relating to an app that you published in GitHub Marketplace.
+- Sponsors webhook is the kind of webhook that subscribe to events related to GitHub Sponsors. 
+- App webhook is a more generic webhook that can be configured to receive specific events that occur in a repository or organization that the app has been granted access to.
+
+In this effort, we will focus on repository webhook because we want to be able to process events happening inside a repository. 
+
+#### Create a repository webhook
+We have a [public project](https://github.com/rjtch/nodejs-webhook-server) on our github-account that is a simple nodejs-server capable to receive webhook-event (But any other server can be used for this end).
+As webhook server we use [Hookdeck](https://hookdeck.com/webhooks/platforms/tutorial-github-webhooks). After everything is well configured we can start the webhook server together with our node-js-server.
```

Comment:
I thought this was about https://github.com/cdevents/webhook-adapter/issues/2.
We should create a plugin for the [webhook adapter](https://github.com/cdevents/webhook-adapter).